policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/cce/nondisabled_young_child_credit_limit.yaml

```
@@ -0,0 +1,9 @@
+description:  Quebec childcare expense credit limit for non-disabled young child (<8 years old).
```

(avoid parameter values in descriptions)
```suggestion
description: Quebec caps childcare expenses at this amount for its childcare expense credit, for young children without disabilities.
```